### ISSUE TITLE
CNN for relation prediction

### ISSUE THREAD

**Issue by lintool** [salman1993/simple-qa-on-kb#9]
We should also build a CNN relation prediction baseline - maybe use Kim CNN as a start?

**Comment by Impavidity** [comment]
I will take this.

**Comment by Impavidity** [comment]
Dataset: valid
no. correct 8961 out of 10845
valid accuracy: 82.627939%
Dataset: test
no. correct 17801 out of 21687
test accuracy: 82.081431%

Hyper-parameters
batch_size = 1000
Adadelta Optimizer, lr = 1.0
multichannel is used
seed = 3435
dev_every = 30
log_every = 10
patience = 50
output_channel = 100
words_dim = embed_dim = 300
dropout = 0.65
word2vec embedding is used

**Comment by salman1993** [comment]
best RNN results for relation prediction on validation set:
Bi-directional GRU - 2 layers - 81.70%

Hyper-parameters
batch_size = 32
Adam Optimizer, lr = 1e-4
seed = 1111
dropout = 0.3
gradient_clip_norm = 0.6
GloVe embedding is used
